### PR TITLE
Fix RealmQuery.isNotEmpty() correctly.

### PR DESCRIPTION
**Pull request by dalinaum**
Fix the native method, Renaming (#2025)

### CODE REVIEW

**Comment by dalinaum** [comment]
Please review this @realm/java It's a patch PR of #2025.

**Comment by cmelchior** [comment]
:+1:

File: changelog.txt [hunk 1]
```diff
@@ -1,13 +1,14 @@
 0.88.0
  * BREAKING CHANGE: DynamicRealm.executeTransaction() now directly throws any RuntimeException instead of wrapping it in a RealmException (#1682).
  * BREAKING CHANGE: DynamicRealm.executeTransaction() now throws IllegalArgumentException instead of silently accepting a null Transaction object.
+ * Added RealmQuery.isNotEmpty() (#2025).
```
Comment (cmelchior) [inline changelog.txt]: Missing acknowledgement

**Comment by stk1m1** [comment]
:thumbsup:

**Comment by kneth** [comment]
:+1:

**Pull request merged**